Client/Client.Qt/Pages/RegistrationPage.cpp

```diff
@@ -61,8 +61,8 @@ RegistrationPage::RegistrationPage(QWidget* parent) : Page(parent)
         if (password != repeatPassword)
         {
             Base::Logger::FileLogger::getInstance().log("passwords are different", Base::Logger::LogLevel::WARNING);
-            _popupSystemMessage->setPopupText("passwords are different");
-            _popupSystemMessage->show();
+            _popupMessage->setPopupText("passwords are different");
```

Comment:
The same as above

Reply:
Dude, WTF? Text is not changed!
Also, you are duplicate text, move it on variable.
And apply namespace Base::Logger::LogLevel to all cpp file. It should become codeless

Reply:
And this recommendation you should apply to ALL code!